api/src/services/assets.ts

```diff
@@ -47,7 +47,7 @@ export class AssetsService {
 		/**
 		 * This is a little annoying. Postgres will error out if you're trying to search in `where`
 		 * with a wrong type. In case of directus_files where id is a uuid, we'll have to verify the
-		 * validity of the uuid ahead of time.
+		 * validity id has type uuid.
```

Comment:
Personally I think the original comment should still be valid (no pun intended), since it never specified it as UUIDv4. But this is a very trivial thing 😄

Reply:
I changed the comment to clarify the context.  I can return the comment to its previous state, as you wish 😏

Reply:
I back comment to original state.

Reply:
Thanks!